libs/market-list/src/lib/utils/market-list.utils.ts

```diff
@@ -20,20 +35,43 @@ export const mapDataToMarketList = ({ markets }: MarketList) =>
       )
       .map((m) => {
         return {
-          id: m.id,
-          decimalPlaces: m.decimalPlaces,
+          ...m,
           marketName: m.tradableInstrument.instrument?.code,
+          settlementAsset:
+            m.tradableInstrument.instrument.product?.settlementAsset?.symbol,
           lastPrice: lastPrice(m) ?? m.data?.markPrice,
           candles: (m.candles || []).filter((c) => c),
+          candleHigh: m.candles
+            ?.reduce(
+              (acc: BigNumber, c) =>
+                c?.high
```

Comment:
Chore (blocking): You'll only loop over 'c' if it's defined, so the optional chaining operator does nothing here. This can simply be 'c.high'. Same for 'c.low' on line 58.

Reply:
I know, but **ts** still complains with 'Object is possibly null'. This is an extra check, I think we can leave this here, don't think this could be blocking.

Reply:
Oh apologies, that should have been 'non-blocking'.